packages/app-store/routing-forms/components/SingleForm.tsx

```diff
@@ -280,6 +289,8 @@ function SingleForm({ form, appUrl, Page, enrichedWithUserProfileForm }: SingleF
       formId: form.id,
       response,
       routeId: route.id,
+      isPreview: true,
+      _enablePerf: searchParams.get("enablePerf") === "true",
```

Comment:
Allows sending enablePerf param by adding query param to route-builder page. Makes it easy to troubleshoot instead of sending fetch requests manually.

Reply:
Nice!